stan/math/prim/mat/fun/to_matrix.hpp

```diff
@@ -2,25 +2,53 @@
 #define STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 
 #include <stan/math/prim/mat/fun/Eigen.hpp>
- // stan::scalar_type
 #include <vector>
 
 namespace stan {
   namespace math {
-
-    // matrix to_matrix(matrix)
-    // matrix to_matrix(vector)
-    // matrix to_matrix(row_vector)
+    /**
+     * Returns the matrix itself.
+     *
+     * @tparam T type of the scalar
+     * @tparam R number of rows
+     * @tparam C number of columns
+     * @param matrix matrix
+     */
     template <typename T, int R, int C>
     inline Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic>
     to_matrix(Eigen::Matrix<T, R, C> matrix) {
```

Comment:
I'm pretty sure this should be `const Eigen::Matrix<T, R, C>&`.  Just make sure the unit tests work.